<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CHEES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartNewGame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Start new game</string>
   </property>
  </widget>
  <widget class="QToolButton" name="EndGame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End game</string>
   </property>
  </widget>
  <widget class="QToolButton" name="LoadGame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Load game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
